components/monitoring/configure_prometheus.yaml

```diff
@@ -0,0 +1,381 @@
+      matchNames:
+      - toolchain-host-operator
+    selector:
+      matchLabels:
+        control-plane: "controller-manager"
+        operators.coreos.com/toolchain-host-operator.toolchain-host-operator: ""
+
+# Member Operator
+- apiVersion: monitoring.coreos.com/v1
+  kind: ServiceMonitor
+  metadata:
+    namespace: ${MONITORING_NAMESPACE}
+    name: sandbox-member-operator
+    labels:
+      prometheus: sandbox-sre
+  spec:
+    endpoints:
+    - bearerTokenSecret:
+        name: ${PROMETHEUS_K8S_SECRET_NAME}
+        key: token
+      interval: 15s
+      scheme: https
+      tlsConfig:
+        insecureSkipVerify: true
+      port: https
+    namespaceSelector:
+      matchNames:
+      - toolchain-member-operator
+    selector:
+      matchLabels:
+        control-plane: "controller-manager"
+        operators.coreos.com/toolchain-member-operator.toolchain-member-operator: ""
+# Loki
+#- apiVersion: monitoring.coreos.com/v1
+#  kind: ServiceMonitor
+#  metadata:
+#    namespace: ${MONITORING_NAMESPACE}
+#    name: loki
+#    labels:
+#      prometheus: sandbox-sre
+#  spec:
+#    endpoints:
+#    - bearerTokenSecret:
+#        name: ${LOKI_SECRET_NAME} # this SA has the permissions to `get /metrics`
+#        key: token
+#      interval: 15s
+#      scheme: https
+#      tlsConfig: 
+#        insecureSkipVerify: true
+#      port: http-metrics
+#    namespaceSelector:
+#      matchNames:
+#      - ${MONITORING_NAMESPACE}
+#    selector:
+#      matchLabels:
+#        app: loki-oauth
+
+# and Prometheus itself
+- apiVersion: monitoring.coreos.com/v1
+  kind: ServiceMonitor
+  metadata:
+    namespace: ${MONITORING_NAMESPACE}
+    name: prometheus
+    labels:
+      prometheus: sandbox-sre
+  spec:
+    endpoints:
+    - bearerTokenSecret:
+        key: ""
+      interval: 15s
+      port: oauth2-proxy
+      scheme: HTTPS
+      tlsConfig: 
+        caFile: /etc/prometheus/tls/tls.crt # volume mounted in the `prometheus` container of the prometheus pods
+        serverName: "prometheus-oauth2.openshift-customer-monitoring.svc"
+    namespaceSelector:
+      matchNames:
+      - ${MONITORING_NAMESPACE}
+    selector:
+      matchLabels:
+        app.kubernetes.io/instance: prometheus
+
+- apiVersion: monitoring.coreos.com/v1
+  kind: PrometheusRule
+  metadata:
+    namespace: ${MONITORING_NAMESPACE}
+    name: deadmanssnitch
+    labels:
+      prometheus: sandbox-sre
+      role: alert-rules
+  spec:
+    groups:
+    - name: deadmanssnitch
+      rules:
+      - alert: SnitchHeartBeat
+        interval: 1m
+        annotations:
+          message: "SnitchHeartBeat: ${CLUSTER_LABEL} has not reported to Deadmanssnitch"
+          runbook: "https://gitlab.cee.redhat.com/service/app-interface/tree/master/docs/app-sre/sop/prometheus/prometheus-deadmanssnitch.md"
+        expr: vector(1)
+        labels:
+          severity: deadman
+
+parameters:
+- name: MONITORING_NAMESPACE
+  required: true
+- name: CLUSTER_LABEL
+  value: sandbox
```

Comment:
What was this used for?

Reply:
this is for the alert message from prometheus as pointer from where the alert is generated. I have updated it to appstudio-stage